dataeng/resources/warehouse-transforms-daily.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -ex
+
+# Setup
+cd $WORKSPACE/warehouse-transforms
+pip install -r tools/dbt_schema_builder/requirements.txt
+
+cd $DBT_PROJECT
+dbt clean
+
+cd $WORKSPACE/warehouse-transforms
+
+dbt run --models tag: $MODEL_TAG --project $DBT_PROJECT --profile $DBT_PROFILE --target $DBT_TARGET --profiles-dir $WORKSPACE/analytics-secure/analytics-tasks/
```

Comment:
I mentioned this before, but I'll mention it again before we run out of runway:   is it possible to not store dbt profiles in the same place we store our pipeline stuff?   Could we start pointing profiles-dir somewhere like $WORKSPACE/analytics-secure/dbt-profiles/?

Reply:
@brianhw It won't be problem for me to change profiles.yaml location to the one you suggested. But i think it would require changes at @bmedx end as well since schema builder task is also using this. 
Also could you explain why would there be a need to have it move in a separate location from our pipeline code!

Reply:
I agree.  This needs a broader discussion. I thought that could happen here, but maybe I'll just bring it up in Slack.